### ISSUE TITLE
Setting ListBox property BorderThickness to 1.5 in XAML throws exception

### ISSUE THREAD

**Issue by wieslawsoltes** [Perspex/Perspex#235]
```
System.FormatException: Nieprawidłowy format ciągu wejściowego.
   w System.Number.ParseDouble(String value, NumberStyles options, NumberFormatInfo numfmt)
   w OmniXaml.TypeConversion.BuiltInConverters.DoubleTypeConverter.ConvertFrom(IXamlTypeConverterContext context, CultureInfo culture, Object value)
   w OmniXaml.ObjectAssembler.ValuePipeline.ConvertValueIfNecessary(Object value, XamlType targetType)
   w OmniXaml.ObjectAssembler.StateCommuter.AssignChildToParentProperty()
   w OmniXaml.ObjectAssembler.Commands.ValueCommand.Execute()
   w OmniXaml.ObjectAssembler.ObjectAssembler.Process(XamlInstruction instruction)
   w OmniXaml.XamlXmlParser.Parse(IEnumerable`1 xamlNodes)
   w OmniXaml.XamlXmlParser.Parse(Stream stream)
   w OmniXaml.XamlLoader.Load(Stream stream)
   w Perspex.Designer.AppHost.PerspexAppHost.<>c__DisplayClass17_0.<DoInit>b__2(Stream stream)
   w Perspex.Designer.AppHost.PerspexAppHost.XamlUpdater()
```

```XAML
            <ListBox BorderThickness="1.5"
                DataContext="{Binding Editor.Project}"
                Items="{Binding Documents}"
                SelectedItem="{Binding Path=CurrentDocument, Mode=TwoWay}"
                Grid.Row="0">
            </ListBox>
```

Setting the property to `1,5` fixes exception. In WPF you use `.` seprarator for floating point values.

The CurrentCulture is `pl-PL`. If I change to `en-US` there no exception, but using `1,5` draws invalid border thickness.

**Comment by grokys** [comment]
PR sumitted to OmniXAML: https://github.com/SuperJMN/OmniXAML/pull/43

**Comment by grokys** [comment]
Closed by 9a8ef44084776302c2f2a967f4da9c510d21833b
